microcloud/cmd/microcloud/ask.go

```diff
@@ -581,7 +581,7 @@ func (c *CmdControl) askNetwork(sh *service.Handler, systems map[string]InitSyst
 	}
 
 	if missingSystems {
-		wantsSkip, err := c.asker.AskBool("Some systems are ineligible for distributed networking. Continue anyway? (yes/no) [default=yes]: ", "yes")
+		wantsSkip, err := c.asker.AskBool("Some systems are missing an interface with no IPs assigned, or a bridge. Continue anyway? (yes/no) [default=yes]: ", "yes")
```

Comment:
I feel this is more ambiguous. It could be understood as "some systems are missing ..., or they are a bridge".
How about:
`Some systems are ineligible for distributed networking (they have neither an interface with no IPs assigned nor a bridge).`
It would be nicer to turn it around ("neither a bridge nor an interface with no IPs assigned"), but we probably want to put more emphasis on the interface with no IPs assigned?

Reply:
The double negative might be confusing as well. 
How is: 
```
Some systems are ineligible for distributed networking: Requires (1) an interface with no IPs assigned or (2) a bridge. Continue anyway? (yes/no) [default=yes]
```

Reply:
```
Some systems are ineligible for distributed networking, which requires either an interface with no IPs assigned or a bridge. Continue anyway? (yes/no) [default=yes]
```
I see why you're using the numbers to make the alternatives clear, but they would prompt me to type either 1 or 2 instead of "yes".